ui/src/app/dashboard/grouped-summary/grouped-summary.component.ts

```diff
@@ -1,49 +1,133 @@
-import {Component, Input} from '@angular/core';
+import {Component, Input, OnInit} from '@angular/core';
 import {JobStatus} from "../../shared/model/JobStatus";
 import {AggregationEntry} from "../../shared/model/AggregationEntry";
 import {Aggregation} from "../../shared/model/Aggregation";
 import {ActivatedRoute} from "@angular/router";
 import {URLSearchParamsUtils} from "../../shared/utils/url-search-params.utils";
+import {JobStatusIcon} from "../../shared/common";
+import {Sort} from "@angular/material";
+
+const LABEL_KEY = 'label';
+const DEFAULT_NUM_ROW = 5;
+
+enum CardStatus {
+  EXPANDED = 'expanded',
+  COLLAPSED = 'collapsed',
+  NONE = 'none'
```

Comment:
I don't think applying a class of "none" really makes sense, I think '' would make more sense here.